terminus/models/block.py

```diff
@@ -19,9 +25,12 @@ def template(cls):
                 </pose>
                 <collision name="{{model.name}}_collision">
                    <geometry>
-                      <box>
-                         <size>95 95 0.15</size>
-                      </box>
+                        <polyline>
+                        {% for vertex in model.vertices %}
+                            <point>{{vertex.x}} {{vertex.y}}</point>
```

Comment:
I think we should later revise using a 3D point when we only use it as a 2D point. Could you please add a comment in the class explaining this and create an issue?